lib/services/auth.js

```diff
@@ -106,6 +106,7 @@ s.authorizeCredential = function (id, authType, scopes) {
 };
 
 s.validateConsumer = function (id, options) {
+  options = options || {};
```

Comment:
I think you can probably do `function(id, options = {})`

Reply:
Good point. Need to remember this! Shall update now and push.